src/dawn/Compiler/Options.inc

```diff
@@ -61,8 +60,8 @@ OPT(std::string, ReorderStrategy, "greedy", "reorder", "",
     "\n - scut   = Use S-cut graph partitioning\n", "<strategy>", true, false)
 OPT(int, MaxFieldsPerStencil, 40, "max-fields", "",
     "Set the maximum number of fields in any given stencils", "<N>", true, false)
-
-
+OPT(bool, PassVerbose, false, "pass-verbose", "",
```

Comment:
what is the best way of doing this?

We currently have quite a mess with our options as we have this as a option for all passes, then we have individual dumps and runtime arguments for some passes. Also I'm not sure if we're really consistent in the use of the F_GROUP in here. Should we clean this up now?

Reply:
can you open up a task then?